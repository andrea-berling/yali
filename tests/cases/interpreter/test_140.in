class Superhero {}
var batman = Superhero();
var superman = Superhero();
// Setting properties on an instance should work
batman.name = "Batman";
batman.called = 90;
// Setting properties on an instance should work
superman.name = "Superman";
superman.called = 43;
// Getting properties on an instance should work
print "Times " + superman.name + " was called: ";
print superman.called;
print "Times " + batman.name + " was called: ";
print batman.called;

