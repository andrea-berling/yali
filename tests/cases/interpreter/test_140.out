Times Superman was called: 
43
Times Batman was called: 
90
